backend/session/session.c

```diff
@@ -262,8 +268,47 @@ static int open_if_kms(struct wlr_session *restrict session,
 	return -1;
 }
 
-static size_t explicit_find_gpus(struct wlr_session *session,
-		size_t ret_len, int ret[static ret_len], const char *str) {
+#if WLR_HAS_FBDEV_BACKEND
+static int open_if_fbdev(struct wlr_session *restrict session,
+		const char *restrict path) {
```

Comment:
This is not necessary. Any checks can be done in the backend on create, and attempt_fbdev_backend can skip things as necessary.

Reply:
My intention was to make this consistent with `open_if_kms` and `attempt_drm_backend`. Why would it be better to do this in `attempt_fbdev_backend` instead?

Reply:
Ah, yes, that's true - I forgot the open_if_kms drmGetVersion check. That's what happens when you try to *remember* things. :)

I believe it would be quite simple to have the checks for both drm and fbdev in their respective `attempt_backend` implementations, in which case we can just calll `wlr_session_open_file` directly and remove `open_if_kms` altogether.